app/models/manager.rb

```diff
@@ -1,15 +1,14 @@
 class Manager < User
   belongs_to :library
-  has_one_attached :image
 
-  enum status:[:active , :inactive]
+  has_one_attached :image
 
-  validates :status ,:email,presence: {message:"This field is required"}
-  validates :status, inclusion: {in: %w(active inactive)}
+  validates :status, :email, presence: { message: 'This field is required'}
+  validates :status, inclusion: { in: %w(active inactive)}
 
-private
+  private
 
-  def issuer?
-    false
-  end
+    def issuer?
+      false
+    end
```

Comment:
Why it is always false?

Reply:
Because I am taking User(parent class) as issuer but not all childs for validation purpose